routes/index.js

```diff
@@ -4,6 +4,8 @@ var router = express.Router();
 var async = require('async');
 var Web3 = require('web3');
 var web3complete = require('web3-complete');
+var nodeVersion = '';
+var blockCount = 128;
```

Comment:
I think maybe the better approach is moving this into the configuration file. Then it is up to the person running it to decide what is a good setting.

Reply:
I've added it as a configuration, this config will also be used when showing an account in order do determine how many blocks too look into to get last transactions for the account.